cc_reader.c

```diff
@@ -281,6 +281,10 @@ int get_token(int c)
 	else if('#' == c)
 	{
 		c = consume_byte(c);
+		while (c == ' ')
```

Comment:
Can we also check for `\t`. GCC allows it and should be easy to add with `||`.

Reply:
Fixed.